src/SDKs/_metadata/network_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-04-26 20:59:33 UTC
+2018-05-17 13:34:00 UTC
```

Comment:
@MikhailTryakhov please generate the sdk using [this](https://github.com/ritwikbasu/azure-sdk-for-net/blob/21e04d84695b939a000037b76757addf664ba8c4/src/SDKs/Network/Management.Network/generate.ps1) script instead